packages/ui/src/stories/modal/DemoModal.svelte

```diff
@@ -14,11 +14,16 @@
 	let modal = $state<Modal>();
 </script>
 
-<Button on:click={() => modal?.show()}>Show</Button>
+<Button
+	onclick={() => {
+		console.log('show');
```

Comment:
Just scanning this PR, noticed this should probably be removed.

Reply:
Done :-)